requirements.txt

```diff
@@ -1,10 +1,13 @@
+cx_Oracle >= 7.1.0
```

Comment:
Similarly, `cc7` available is `5.2.1-1.el7.cern`

Reply:
Available version in OS works.  So I should remove the requirement or specify `>=` the version available in cc7?

Reply:
Again, I'd leave no version restriction per-se (here the `RPM` package name is as written)